docs/quick-start/getting-started.md

```diff
@@ -16,6 +16,12 @@ Once installed, you need to add the Reactotron client to your project. Follow on
 - [**React Native**](./react-native.md#installing-reactotronapp)
 - [**React**](./react-js.md#installing-reactotronapp)
 
+If you run into an [issue](https://github.com/infinitered/reactotron/issues/1140) opening the application on Mac you can install with Homebrew instead. 
+
+```
+brew cask install reactotron
+```
```

Comment:
I'm not entirely sure we want to point people to use `brew` because I believe the last release is still 2.0

Would be interesting to add brew publishing back 👀

Reply:
Wait someone updated it 🤯 
- [How to Create and Maintain a Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap)
- [New Reactotron tap](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/r/reactotron.rb)
- [Commit to update to 3.1.0](https://github.com/Homebrew/homebrew-cask/commit/c716df6134d0b842f8dcc23644131d737604c716)